### DISCUSSION TITLE
Mobile app

### DISCUSSION THREAD

**Discussion by ghost** [stoatchat/discussions#150] [Feature Suggestions]
This is a bit of a huge suggestion, a large task to undertake, but a mobile app would be cool.

**Comment by sussycatgirl** [comment]
An Android app is currently being tested I think, and a few people (including me) wanted to make their own apps too.

**Reply by sparkle8205** [reply]
will there be an iOS app?

**Reply by Rexogamer** [reply]
@Zomatree is developing the iOS app; @infi is developing the official Android app.

**Reply by tjb** [reply]
@Rexogamer Would it be possible to contribute to the iOS app?
